packages/core/src/App/Containers/WarningScamMessageModal/fake-message-content.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { getBrandName } from '@deriv/shared';
+
+const brandName = getBrandName();
+
```

Comment:
maybe it would be more correct to define brandName inside the component as it is used there? 
if so, please check also in other files